frontend/packages/component-library/README.md

```diff
@@ -59,3 +59,9 @@ You can run the tests using the following commands:
    ```bash
    yarn lint
    ```
+   
+## Linking (making changes to component-library and seeing them in another project)
```

Comment:
I think adding it to the `apps/README.md` may make more sense, running `yarn dev` is implied in this context.